src/main/java/org/gridsuite/ds/server/service/DynamicSimulationRunContext.java

```diff
@@ -24,14 +28,27 @@ public class DynamicSimulationRunContext {
 
     private final byte[] dynamicModelContent;
 
-    public DynamicSimulationRunContext(UUID networkUuid, String variantId, int startTime, int stopTime, byte[] dynamicModelContent) {
+    private final byte[] eventModelContent;
+
+    private final byte[] curveContent;
+
+    private final DynamicSimulationParameters parameters;
+
+    public DynamicSimulationRunContext(String receiver, UUID networkUuid, String variantId, int startTime, int stopTime, byte[] dynamicModelContent, byte[] eventModelContent, byte[] curveContent, DynamicSimulationParameters parameters) {
+        this.receiver = receiver;
         this.networkUuid = Objects.requireNonNull(networkUuid);
         this.variantId = variantId;
         this.startTime = startTime;
```

Comment:
Is it still relevant to carry start and stop as you also have parameters (DynamicSimulationParameters) which contains theses attributes?

Reply:
Yes! start and stop times have been removed. These params have been now set earlier at the beginning of the run..